external/fv3fit/fv3fit/keras/_models/precipitative.py

```diff
@@ -400,6 +385,5 @@ def predictor(self) -> PureKerasModel:
             self.input_variables,
             # predictor has additional diagnostic outputs which were indirectly trained
             list(self.output_variables),
-            get_output_metadata(self.output_packer, SAMPLE_DIM_NAME),
```

Comment:
Don't you need to provide `unstacked_dims = ('z',)` here?

Reply:
Oh I see, `unstacked_dims` has that as a default. 

Nit: Can the calls of `PureKerasModel` be consistent in whether they provide this argument (e.g, in the dense model it does but here it doesn't)? Would make the intended usage clearer. Maybe don't have a default for unstacked_dims so that people reading the code will see what's passed for them to help understand what's happening?

Reply:
Great suggestion. I made unstacked_dims a required argument.